logging/cluster-logging-loki.adoc

```diff
@@ -1,23 +1,30 @@
 :_content-type: ASSEMBLY
 :context: cluster-logging-loki
 [id="cluster-logging-loki"]
-= Loki
+= Logging using LokiStack
 include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/cluster-logging-loki-about.adoc[leveloffset=+1]
+In {logging} documentation, _LokiStack_ refers to the {logging} supported combination of Loki, and web proxy with {product-title} authentication integration. LokiStack's proxy uses {product-title} authentication to enforce multi-tenancy. _Loki_ refers to the log store as either the individual component or an external store.
```

Comment:
With the comma before "and" I can't make out the meaning of the first phrase; I suspect the comma should not be there. Also, "web proxy" is singular and requires "a" or "the". Also, maybe "The LokiStack proxy" not "LokiStack's proxy"? (Not sure about that last one).